test/unit/sql/where.test.js

```diff
@@ -423,6 +423,37 @@ suite(Support.getTestDialectTeaser('SQL'), function() {
             });
           });
         });
+
+        suite('$like', function() {
+          testsql('userId', {
+            $like: {
+              $any: ['foo', 'bar', 'baz']
+            }
+          }, {
+            postgres: "\"userId\" LIKE ANY (ARRAY['foo','bar','baz'])"
+          });
+          testsql('userId', {
+            $iLike: {
+              $any: ['foo', 'bar', 'baz']
+            }
+          }, {
+            postgres: "\"userId\" ILIKE ANY (ARRAY['foo','bar','baz'])"
+          });
+          testsql('userId', {
+            $notLike: {
+              $any: ['foo', 'bar', 'baz']
+            }
+          }, {
+            postgres: "\"userId\" NOT LIKE ANY (ARRAY['foo','bar','baz'])"
```

Comment:
I think the query generator adds them, but I don't think Postgres requires them.